docker/pravega/entrypoint.sh

```diff
@@ -26,12 +26,36 @@ configure_controller() {
     echo "JAVA_OPTS=${JAVA_OPTS}"
 }
 
-configure_segmentstore() {
-    add_system_property "pravegaservice.zkURL" "${ZK_URL}"
-    add_system_property "autoScale.controllerUri" "${CONTROLLER_URL}"
+configure_nfs_s3_hdfs() {
```

Comment:
I'm missing the documentation for how to use this. If it is going to be done separately, then we need a subtask for issue #1336 .